### ISSUE TITLE
[Bug] Error: Unsupported source map encoding charset=utf8;base64

### ISSUE THREAD

**Issue by ericnoguchi** [sasstools/scss-tokenizer#13]
I using gulp sass and  In node_modules\scss-tokenizer\lib\previous-map.js
the below function breaks when encoding is data:application\/json;charset=utf8;base64

This issue stops the whole build process
```
    PreviousMap.prototype.decodeInline = function decodeInline(text) {
        var uri = 'data:application/json,';
        var base64 = 'data:application/json;base64,';

        if (this.startWith(text, uri)) {
            return decodeURIComponent(text.substr(uri.length));
        } else if (this.startWith(text, base64)) {
            return _jsBase.Base64.decode(text.substr(base64.length));
        } else {
            var encoding = text.match(/data:application\/json;([^,]+),/)[1];
            throw new Error('Unsupported source map encoding ' + encoding);
        }
    };
```

**Maybe a fix similar to https://github.com/postcss/postcss/commit/215ed88db2e6e180f7f07750ddc1540de4ad25d2**

**Comment by ericnoguchi** [comment]
Hi,

Would you like me to submit a pull request for this issue? it is a bit urgent as we now have manually updated the file when doing npm install.

**Comment by xzyfer** [comment]
Pull requests always welcome. I'm.kn holiday so I haven't looked at the
issue.

**Comment by yavin5** [comment]
@ericnoguchi Could you please show us what you changed to fix it?

**Comment by ericnoguchi** [comment]
@yavin5 

For now I manually used the same fix for postcss/postcss@215ed88 see my first comment, you will have to do this again if you  remove node_modules and reinstall.
